app/controllers/transfer_packages_controller.rb

```diff
@@ -17,16 +72,54 @@ def find_sample
 
   private
 
+  def transfer_package_params
+    params.require(:transfer_package).permit(
+      :receiver_institution,
+      :recipient,
+      :includes_qc_info,
+    ).tap do |whitelisted|
```

Comment:
`whitelisted` => `allowed`

But with the proposed simplification of `#create` you wouldn't need any manipulation of the params.